class-udm-updater.php

```diff
@@ -43,6 +43,8 @@ class Updraft_Manager_Updater_1_8 {
 	 */
 	public function __construct($mothership, $muid, $relative_plugin_file, $options = array()) {
 
+		global $wp_version;
```

Comment:
@andtown Unfortunately there are some plugins that over-write `$wp_version` so that the "Generator" meta-tag on the front-end gives a false value, which they wrongly believe will help security. For that reason, if you want to use `$wp_version`, you should do an `include()` on `ABSPATH.WPINC.'/version.php'`.

Reply:
@DavidA Thanks for letting me know about this, I'm wondering what threat could possibly be prevented by changing `$wp_version`. I've included the `version.php` variable in the constructor method.